Document selective no_ssl_verification setting

srenatus opened the pull request:
The change (SPOOL-486) hasn't been merged yet, but this is what it's
going to look like.

jamescott commented:
and also this:

https://github.com/chef/chef-docs/commit/f6d4c48c740e25412f5a2abe985cdb4fc70673d2

srenatus committed:
Document selective no_ssl_verification setting

The change (SPOOL-486) hasn't been merged yet, but this is what it's
going to look like.

Signed-off-by: Stephan Renatus <srenatus@chef.io>

Commit:
Copy edits

kagarmoe commented:
![gif-keyboard-6535629795279786763](https://cloud.githubusercontent.com/assets/4400151/23049028/65bc9926-f46e-11e6-8b48-791b5acd8ef9.gif)

srenatus commented:
@kagarmoe -- I'm glad this was merged, but the feature hasn't shipped yet. Hope that's not too bad.
